src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__stars {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  & :nth-child(-n + 4) {
```

Comment:
The selector '& :nth-child(-n + 4)' has a space between '&' and ':nth-child', which might cause unexpected behavior. It should be '&:nth-child(-n + 4)' to correctly apply styles to the first four child elements.